component/main.tsx

```diff
@@ -16,3 +16,9 @@ const config = {
 firebase.initializeApp(config);
 
 ReactDOM.render(<ClientCore />, document.getElementById("root"));
+
+const storageService = (chrome.extension.getBackgroundPage() as any).BackgroundStorageService;
```

Comment:
Store as in the Redux Store?
This is how you retrieve the service from background.js